src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/hardware/Computer.kt

```diff
@@ -8,4 +8,8 @@ abstract class Computer internal constructor() {
     internal abstract val instanceType: InstanceType
 
     internal abstract fun setUp(ssh: SshConnection)
+
+    var volumeSize = 100
+
+    fun withVolumeSize(volumeSize : Int) = apply { this.volumeSize = volumeSize }
```

Comment:
I'm not sure if it's not too specific. AFAIR volume has at least 3 parameters (size, IOPS, type). 
```
withVolume(volume : Volume)`

class Volume(val size);
``

It could be more open to extending in the future.